src/objects/Sprite.js

```diff
@@ -101,9 +101,18 @@ Sprite.prototype = Object.assign( Object.create( Object3D.prototype ), {
 		return function raycast( raycaster, intersects ) {
```

Comment:
It would pass the camera as an additional parameter and avoid any modifications to `Raycaster`.

Reply:
If you change the signature, then you have to modify the Raycaster which calls it. Right?

Reply:
I totally forgot that the method invocation happen via `Raycaster`. Too bad, I originally wanted to avoid a tight coupling between `Raycaster` and `Camera`...

Anyway, in this case my comment is of course obsolete.